src/neo4j_graphrag/tools/tool.py

```diff
@@ -211,23 +211,28 @@ def validate_properties(self) -> "ObjectParameter":
 class Tool(ABC):
     """Abstract base class defining the interface for all tools in the neo4j-graphrag library."""
 
+    _name: str
```

Comment:
Do you remember why we need to have these declarations here? Mypy stuff?

Reply:
No idea :)